### PR TITLE
Add CHANGELOG entry for 2191, update task to use cumuluss project

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** 
Addresses [CUMULUS-2191] (https://bugs.earthdata.nasa.gov/browse/CUMULUS-2191)

Adds CHANGELOG, updates image to point to cumuluss/async-operation:28 dockerhub project

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,9 +19,24 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
+    - Removed the following from the `@cumulus/api/models.asyncOperation` class in
+      favor of the added `@cumulus/async-operations` module:
+      - `start`
+      - `startAsyncOperations`
+
 ### Added
 
 - **CUMULUS-2185** - RDS Migration Epic
+  - **CUMULUS-2191**
+    - Updated cumuluss/async-operation task to write async-operations to the RDS
```
Comment (markdboyd) [inline CHANGELOG.md:35]: This seems like it belongs under an "Updated" header?

Comment (Jkovarik) [reply CHANGELOG.md:35]: Hmm, given it has 'updated' in it... probably 😆

Comment (Jkovarik) [reply CHANGELOG.md:35]: 61018962f

Comment (markdboyd) [reply CHANGELOG.md:35]: And removed from here?

**Review by markdboyd — APPROVED** [review]

**Pull request merged**